### DISCUSSION TITLE
Innernet on Synology DiskStation

### DISCUSSION THREAD

**Discussion by rschulman** [tonarino/innernet#220] [Show and tell]
I got innernet running on a Synology DiskStation NAS device, so I can access file shares from devices even if I'm not on my home LAN. It wasn't terribly hard, but it wasn't straight forward either. I thought others might be in a similar situation, so I wrote up my results here: https://rbs.io/2022/05/wireguard-and-innernet-on-synology/

I hope its helpful for others!

-Ross Schulman

**Comment by strohel** [comment]
Hi @rschulman, thanks for sharing this, I'm sure some people will be grateful for it!

**Comment by mcginty** [comment]
Thanks @rschulman, nicely written article! If anyone wants to write an innernet synology app, please feel free to email me and I'm happy to help if they get stuck anywhere.
